### PR TITLE
Move CSN space to prevent overflow

### PR DESCRIPTION
**Pull request by bettse**
# What's new

- Fix for https://github.com/flipperdevices/flipperzero-firmware/issues/2185
- Removes space between colon and first CSN byte.  Should allow enough room to fit a full 00's CSN.

# Verification 

- You could try to reproduce the bug and see it is fixed
- Alternatively just see the space has been moved and estimate that it'll correct the problem from the bug

# Checklist (For Reviewer)

- [x] PR has description of feature/bug or link to Confluence/Jira task
- [x] Description contains actions to verify feature/bugfix
- [x] I've built this code, uploaded it to the device and verified feature/bugfix

### CODE REVIEW

**Comment by bettse** [comment]
cc @pcunning

File: applications/plugins/picopass/scenes/picopass_scene_read_card_success.c [hunk 1]
```diff
@@ -33,7 +33,7 @@ void picopass_scene_read_card_success_on_enter(void* context) {
     uint8_t csn[PICOPASS_BLOCK_LEN];
     memcpy(csn, &AA1->data[PICOPASS_CSN_BLOCK_INDEX], PICOPASS_BLOCK_LEN);
     for(uint8_t i = 0; i < PICOPASS_BLOCK_LEN; i++) {
-        furi_string_cat_printf(csn_str, " %02X", csn[i]);
+        furi_string_cat_printf(csn_str, "%02X ", csn[i]);
```
Comment (skotopes) [inline applications/plugins/picopass/scenes/picopass_scene_read_card_success.c:36]: Nope, not going to fix issue ;-) Space is still space and it's handled when rendering. But even if you trim last space there will be not enough space to fix the line.

**Review by skotopes — CHANGES_REQUESTED** [review]

**Comment by skotopes** [comment]
Is it ok if we remove spaces between?

**Comment by bettse** [comment]
Sorry for the delay, the idea was that the space _sometimes_ was too long, but since it doesn't wrap (as in the failure example) it would be offscreen whitespace.  Inelegant.  I'm 100% in favor or just removing the spaces.  I'll try to get around to that, but feel free to do it and close this out if I'm too slow.

**Review by skotopes — APPROVED** [review]

**Pull request merged**